src/main/java/com/codecool/virtualpub/data/Cutomer.java

```diff
@@ -61,6 +61,16 @@ public void Drink(int alcoholUnit) {
         this.alcoholLevel += alcoholUnit;
     }
 
+    public String Speak() {
+        Random random = new Random();
```

Comment:
I would avoid creating a new Random instance every time this function is called. I would store a Random instance as a field of the Customer class and use that.